pipeline.yml

```diff
@@ -37,7 +37,7 @@ resources:
   type: git
   source:
     uri: https://github.com/18F/cg-compliance.git
-    branch: master
+    branch: update2
```

Comment:
I actually forgot to push the change. It should stay master. update2 is for testing.